app/controllers/editorial/nodes_controller.rb

```diff
@@ -27,9 +27,7 @@ def prepare
       configure_defaults!
       authorize! :create_in, @section
 
-      @node_types = Node.descendants.reject {|klass|
-        [RootNode, SectionHome].include? klass
-      }.map do |klass|
+      @node_types = [GeneralContent].map do |klass|
```

Comment:
I was running under the assumption that we would add more types at some stage so I didn't want to remove the whole prepare page/selector thing.